addons/l10n_pe/i18n/l10n_pe.pot

```diff
@@ -4,246 +4,9576 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 14.0+e\n"
+"Project-Id-Version: Odoo Server 15.0+e\n"
```

Comment:
`.pot` is useless while this module is not set in Transifex, even more if you don't update the relevant `.po` files